vhdl_lang/src/ast.rs

```diff
@@ -1375,6 +1378,10 @@ pub struct ConfigurationSpecification {
     pub vunit_bind_inds: Vec<VUnitBindingIndication>,
 }
 
+/// LRM 7.4 Disconnection specification
+#[derive(PartialEq, Debug, Clone)]
+pub struct DisconnectionSpecification {}
```

Comment:
Here, we should put all of the AST elements. You are currently parsing this very nicely, but then drop all of the information. This should include all relevant AST elements, such as 
- The ident list
- The type
- The wait expression

Reply:
This should be fixed now. The only thing I couldn't do was multiple idents. I left a little TODO for that. I am also not quite sure what ident (if any) to return when it says Others or All. Maybe this needs to be done in the analyzer or so.

Reply:
Done, see https://github.com/VHDL-LS/rust_hdl/pull/307#discussion_r1625150504